gamma_bridge/exceptions.py

```diff
@@ -20,3 +20,23 @@ class GammaConnectionError(GammaException):
     def __init__(self, message=None, *args):
         self.message = "External connection error to the rg-gamification-traking application. {}".format(message)
         super(GammaConnectionError, self).__init__(self.message, *args)
+
+
+class GammaValueError(GammaException, ValueError):
+    """
+    Base class for Gamma ValueError.
+    """
+
+    def __init__(self, message=None, *args):
+        self.message = "Wrong type of value. {}".format(message)
+        super(GammaException, self).__init__(message, *args)
+
+
+class NotBaseStorageError(GammaValueError):
+    """
+    Error related to work with BaseStorage type.
+    """
+
+    def __init__(self, message=None, *args):
+        self.message = "The object doesn't have a BaseStorage type or children of last. {}".format(message)
+        super(GammaException, self).__init__(message, *args)
```

Comment:
In this case, `self.message` will be overwritten in the superclass.
So it will be `"Wrong type of value. Current type: StorageLogger"`

Reply:
I  have fixed the issues with the `message`.